entrypoint.sh

```diff
@@ -208,7 +212,18 @@ create_runner () {
         # Check if instance creation failed
         if [ -z "$INSTANCE_ID" ]; then
             debug "Failed to create instance with bid price ${BID_PRICE}"
-            BID_PRICE=$(echo "$BID_PRICE * 1.1" | bc)
+            # if bid price is too low, and the error message contains"An error occurred (SpotMaxPriceTooLow) when calling the RunInstances operation: Your Spot request price of 0.0035200000000000023 is lower than the minimum required Spot request fulfillment price of 0.06319999999999999."
```

Comment:
from code spec we can have this exception catch, but should we move this code comments into FAQ as a part of readme?
I am not sure if there any user/use case want to break if the price is ... higher than 0.0035200000000000023 ?

Reply:
the two way of adjusting bid prices are here.  If we have more heuristics, I feel then we should factor this code and support a more generic algorithm.

Reply:
new issue created accordingly/